### PR TITLE
healthcare API: update remaining dicom/ and hl7v2/ files from v1beta1 to v1

### PR DESCRIPTION
**Pull request by noerog**
NOTE: This PR is fundamentally a branch of https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/healthcare/v1beta1 to v1. The code has been previously reviewed in this repository.

The only real code change is in healthcare/v1/src/main/java/snippets/healthcare/hl7v2/Hl7v2StorePatch.java which now uses the "Hl7v2NotificationConfig" class instead of the "NotificationConfig" class.

After all healthcare/v1 samples are approved, I'll delete the existing v1beta1/ directory.

### CODE REVIEW

**Comment by noerog** [comment]
Sorry for the large PR, but this is basically just a branch from one existing directory to another one with the primary change being "v1beta1" -> "v1" and the one change I listed in the parent comment.

File: healthcare/v1/src/main/java/snippets/healthcare/dicom/DicomStoreImport.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    // Create request and configure any parameters.
+    DicomStores.CloudHealthcareImport request =
+        client
+            .projects()
+            .locations()
+            .datasets()
+            .dicomStores()
+            .healthcareImport(dicomStoreName, importRequest);
+
+    // Execute the request, wait for the operation to complete, and process the results.
```
Comment (gguuss) [inline healthcare/v1/src/main/java/snippets/healthcare/dicom/DicomStoreImport.java:63]: Copy / paste nit, duplicate comment

File: healthcare/v1/src/main/java/snippets/healthcare/dicom/DicomStoreList.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2019 Google LLC
```
Comment (gguuss) [inline healthcare/v1/src/main/java/snippets/healthcare/dicom/DicomStoreList.java:2]: If it's a new file, 2020.

Comment (noerog) [reply healthcare/v1/src/main/java/snippets/healthcare/dicom/DicomStoreList.java:2]: The OSS team said that if you're just moving a file, then you keep the original copyright date. I don't know the particulars of whether copying a file into a new directory constitutes "moving", wdyt?

File: healthcare/v1/src/main/java/snippets/healthcare/dicom/DicomStoreImport.java (hunk 1 above)
Comment (noerog) [reply healthcare/v1/src/main/java/snippets/healthcare/dicom/DicomStoreImport.java:63]: Done.

**Pull request merged**